docs/docs.sh

```diff
@@ -0,0 +1,149 @@
+#!/bin/bash
+
+# Copyright 2020-2022, RespiraWorks
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+OUTPUT_DIR=_build
+
+# Fail if any command fails
+set -e
+set -o pipefail
+
+# Print each command as it executes
+if [ -n "$VERBOSE" ]; then
+  set -o xtrace
+fi
+
+# This script should work no matter where you call it from.
+cd "$(dirname "$0")"
+
+EXIT_FAILURE=1
+EXIT_SUCCESS=0
+
+# Check if Linux
+PLATFORM="$(uname -s)"
+if [ $PLATFORM != "Linux" ]; then
+  echo "Error: This script only supports 'Linux'. You have $PLATFORM."
+  exit $EXIT_FAILURE
+fi
+
+print_help() {
+    cat <<EOF
+Utility script for generating Ventilator documentation.
+
+The following options are available:
+  install     One-time installation of build toolchain and dependencies
+  clean       Clean build directories
+  build       Generate documentation
+  view        Open generated documentation in browser
+  help/-h     Display this help info
+EOF
+}
+
+clean_all() {
+  clean_dir ${OUTPUT_DIR}
+  # \todo clean intermediate files from wiring and parts
+}
+
+clean_dir() {
+  dir_name=$1
+  if [ -d "$dir_name" ]; then
+    echo "Removing $dir_name"
+    rm -rf "$dir_name"
+    return 0
+  elif [ -f "$dir_name" ]; then
+    echo "File with this name already exists, not a directory."
+    return 1
+  fi
```

Comment:
Any return value if no file or directory exists with that name? Doesn't really matter but it seems weird.

Reply:
I think in that case, the called `rm` command should return an error.